### PR TITLE
feat(FXA-54): add outcomes

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes https://github.com/mozilla/fxa/issues/144

### CODE REVIEW

**Comment by vladikoff** [comment]
@rfk r?

File: features/FxA-54-push-on-status-change/README.md [hunk 1]
```diff
@@ -30,11 +30,26 @@ As a concrete metric
...
 
+## Outcomes
+
+We have decreased the polling rate in Firefox Desktop and now rely on Push to provide quick
+feedback to the user about their account state. The initial verification timeout was increased
+from 5 seconds to 15 seconds. Subsequent timeout was extended from 15 seconds to 30 seconds.
+This will now allow Firefox Desktop to use less server resources and still provide good user experience.
+
+From our metrics (10% sample rate) we gathered that ~5% of users use a push notification from
+the auth server to verify their account in desktop.
```
Comment (rfk) [inline features/FxA-54-push-on-status-change/README.md]: What's the denominator in this percentage?  Is it number of `/v1/recovery_email/status` calls with `reason=push` as a percentage of all `/v1/recovery_email/status` calls?

Comment (rfk) [inline features/FxA-54-push-on-status-change/README.md]: (Actually no, it can't possible be that as it's far too big a percentage...)

Comment (vladikoff) [inline features/FxA-54-push-on-status-change/README.md]: From above: >We will compare the number of notifications sent and number of verifications used by Desktop.

so `used (http status) / sent (from the app logs)`

Comment (rfk) [inline features/FxA-54-push-on-status-change/README.md]: So 95% of notifications sent, did *not* result in the browser querying email status with `reason=push`?

Comment (vladikoff) [inline features/FxA-54-push-on-status-change/README.md]: Yeah, which means users are slow to verify or email polling took care of it.

**Comment by rfk** [comment]
@vladikoff LGTM, thanks!  I slightly re-worded the explanation of the graph based on your comments here, and moved the card into "shipped".

**Comment by vladikoff** [comment]
Thanks Ryan!

**Pull request merged**